src/dev/utility/configuration.cpp

```diff
@@ -93,7 +93,6 @@ bool configuration::load(const char* file_name, const char* arguments)
     if (arguments != nullptr)
     {
         std::string str_arguments(arguments);
-        std::replace(str_arguments.begin(), str_arguments.end(), ',', ';');
```

Comment:
@qinzuoyan I cannot remember exactly what it is for, should be related to something you told before? I have to remove this here for the purpose of enabling options like "toollets=tracer,profiler;port=34801" for -cargs.